static/src/javascripts/lib/closest.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+/**
+ * Adds a wrapper around Element.closest()
+ */
+
+function closest(element: ?Element, selectors: string): ?Node {
+    if (!element) return null;
+
+    if ('closest' in Element.prototype) {
+        return element.closest(selectors) || null;
+    }
+
+    let el = element.parentElement;
```

Comment:
I think I had to test for nodeType because otherwise it threw on `document.matches` when it goes all the way up the tree.